File: testdata/AnnotatedMethodParameters.java
import javax.annotation.Resource;

class AnnotatedMethodParameters
{
  void a
  (
    @Resource AnnotatedMethodParameters that,
    int ordinaryParam    
  )
  {
    System.out.println(that)  
  }

  void b
  (
    float f,
    @javax.annotation.Resource AnnotatedMethodParameters that,
    int ordinaryParam    
  )
  {
    System.out.println(that)  
  }

  
  void c
  (
    float f,
    @Resource AnnotatedMethodParameters that,
    @Resource(name = "hi") String foo
  )
  {
    System.out.println(that)  
  }  
}

File: testdata/expected/default/AnnotatedMethodParameters.scala
import javax.annotation.Resource

class AnnotatedMethodParameters
{
  def
  a
  (
    @Resource that : AnnotatedMethodParameters,
    ordinaryParam : Int
  ) : Unit =
  {
    System.out.println(that)  
  }

  def
  b
  (
    f : Float,
    @javax.annotation.Resource that : AnnotatedMethodParameters,
    ordinaryParam : Int
  ) : Unit =
  {
    System.out.println(that)  
  }

  
  def
  c
  (
    f : Float,
    @Resource that : AnnotatedMethodParameters,
    @Resource(name = "hi") foo : String
  ) : Unit =
  {
    System.out.println(that)  
  }  
}